test/unit/mongo_client.test.js

```diff
@@ -1,4 +1,6 @@
-'use strict';
+import { inspect } from 'util';
+
+('use strict');
```

Comment:
nit: should be the first line

Reply:
Yeah, I was intrigued that the linter didn't have a problem with this, but I guess it sees it as a programmer judgement call.